client/src/SolrIndexer.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { describe, expect, it } from "@jest/globals";
+import { mount, shallow } from "enzyme";
+import toJson from "enzyme-to-json";
+import SolrIndexer from "./SolrIndexer";
+
+describe("SolrIndexer", () => {
+    it("renders", () => {
+        const wrapper = shallow(<SolrIndexer />);
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("changes the pid input", () => {
+        const wrapper = mount(<SolrIndexer />);
+        wrapper.find("#solrIndexPid").simulate("change", {
```

Comment:
Same comment here as for PdfGenerator.test.js.

Reply:
refer to previous comment.